autoload/asyncomplete/sources/flow.vim

```diff
@@ -1,3 +1,4 @@
+let s:iswin = has('win32') || has('win64')
```

Comment:
should be `s:is_win` to be consistent with the usage in `s:resolve_flowbin_path`